### PR TITLE
IIS Task Win10 Fix

### PR DESCRIPTION
**Pull request by mvvsubbu**
In Win10 extra quotes causing problem.. removing that causing problem in
other OS. So handling it with if else.

### CODE REVIEW

File: Tasks/IISWebAppDeployment/MsDeployOnTargetMachines.ps1 [hunk 1]
```diff
@@ -55,14 +55,29 @@ function ThrowError
...
+    $osMajorVer = $matches[0].Groups[1].Value    
+    Write-Verbose "Current OS Version : $osMajorVer" -Verbose
+
+    if( $osMajorVer -eq 10)
+    {        
+        $result = cmd.exe /c "$command"    
+    }
+    else
+    {
+        $result = cmd.exe /c "`"$command`""
```
Comment (kmkumaran) [inline Tasks/IISWebAppDeployment/MsDeployOnTargetMachines.ps1]: Is this due to Win10 or PS version?

File: Tasks/IISWebAppDeployment/MsDeployOnTargetMachines.ps1 [hunk 2]
```diff
@@ -55,14 +55,29 @@ function ThrowError
 function Run-Command
```
Comment (kmkumaran) [inline Tasks/IISWebAppDeployment/MsDeployOnTargetMachines.ps1]: Is this applicable in other script of ours?

**Comment by mvvsubbu** [comment]
Yes .. it is applicable for SQL related tasks as well. We should fix them ..

**Comment by mvvsubbu** [comment]
It should be because of PsVersion

**Comment by kmkumaran** [comment]
Looks fine. As discussed, let's follow-up with PowerShell team to understand the right way to handle this.

**Comment by mvvsubbu** [comment]
Sure will do

**Pull request merged**